### PR TITLE
Add config key to change batch size

### LINKED ISSUE #206: Allowed memory size exhausted during search:import
I'm trying to `search:import` my products (37 000) and I got this error : 

    php bin/console search:import --indices=product

    PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 12288 bytes) in .../vendor/doctrine/dbal/lib/Doctrine/DBAL/Types/ArrayType.php on line 58
    Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 12288 bytes) in .../vendor/doctrine/dbal/lib/Doctrine/DBAL/Types/ArrayType.php on line 58
    PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 32768 bytes) in .../vendor/symfony/debug/Exception/OutOfMemoryException.php on line 1
    Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 32768 bytes) in .../vendor/symfony/debug/Exception/OutOfMemoryException.php on line 1

I'm using `algolia/search-bundle": "^3.0`

To much entries ?

### PR DESCRIPTION
**Pull request by julienbourdeau**
The import command also follows this config option.

Fix #206

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+3.1.0
+-----
+
+- Introduce `batchSize` config key (default: 500) - PR [#208](https://github.com/algolia/search-bundle/pull/203)
+    
+    This config allow you craete smaller or later batch call Algolia. The same config is used for the doctrine in the ImportCommand
```
Comment (tristanbes) [inline CHANGELOG.md]: later -> bigger
craete -> create
the doctrine -> doctrine

File: src/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -31,6 +31,9 @@ public function getConfigTreeBuilder()
                 ->scalarNode('settingsDirectory')
                     ->defaultValue(null)
                 ->end()
+                ->scalarNode('batchSize')
```
Comment (alcaeus) [inline src/DependencyInjection/Configuration.php]: Unless you're planning to reuse the batch size for other operations as well, I'd suggest naming this `reindexBatchSize` to better document its use in the name.

File: CHANGELOG.md (hunk 1 above)
Comment (julienbourdeau) [reply CHANGELOG.md]: Thanks! Updated all the typos.

File: src/DependencyInjection/Configuration.php (hunk 2 above)
Comment (julienbourdeau) [reply src/DependencyInjection/Configuration.php]: It's also used in any `index()` call, like `$indexManager->index($threeThousandObjects)` will split the data into chuck of 500 objects.

**Comment by julienbourdeau** [comment]
@alcaeus Thanks for the advice! I'll add a clear call right way 👍

File: src/DependencyInjection/Configuration.php (hunk 2 above)
Comment (julienbourdeau) [reply src/DependencyInjection/Configuration.php]: And it's used in `$indexManager->remove($threeThousandObjects)` as well. That's why I'd rather keep `batchSize`.

**Pull request merged**